packages/fxa-auth-db-mysql/docs/API.md

```diff
@@ -2543,20 +2544,18 @@ Content-Length: 2
 ### Example
 
 ```
-
```

Comment:
Why am I messing with the whitespace here? The extra blank lines inside the pre-formatted blocks caused me to misread which block of text they applied to further down and add extra backticks unnecessarily. I think it's clearer this way.

Reply:
Will prettier undo this later, or re-add it?

Reply:
It doesn't seem to, for me.